### ISSUE TITLE
Add documentations for heach hive

### ISSUE THREAD

**Issue by Zykino** [muesli/beehive#300]
I’m trying to use those hives but I’m not clear on how to do it. It will be clearer if hive maker added documentation on how to configure them. A (formatted?) text explaining how to configure the hive (generate secret keys, …) with a link to the official website for example.

For example:
Discord, should I create an application or a bot?
Twitter, I need to give my phone number?
Twitch, I tried to configure something, and it got totally disabled. But I don’t know AT ALL where to find the keys.
Mastodon, I think it is working but, a bit of doc would make me confident about it.

I would love this kind of documentation in the admin interface, when clicking on a hive.

On the other hand the documentation here on what each hive can do is already really nice (for a dev at least). The list of, configurations, events, and action is nice at it is, at least for the hive I tried / needed.

**Comment by muesli** [comment]
I agree, we desperately need some help here. Ideally we start writing these docs as markdown, so we can easily generate html & plaintext formats and benefit in multiple places (like a website and the admin interface) from it.

**Comment by Zykino** [comment]
So I started creating some notes/descriptions. They may note be clear enough since I wrote them once I got the bee and chain working (mostly). https://wiki.zykino.net/StreamDeck

I will continue to add some here but feel free to use it or ask me my current notes if the address changed.
